tests/conftest.py

```diff
@@ -57,8 +75,8 @@ def mock_config():
 @pytest.fixture
 def mock_gridded_obs_dataset():
```

Comment:
I'd call this "sample_gridded_ob_dataset"... "mock" means something else (you'd typically mock interfaces and actions, but this is very much just sample data).